xml_inscriptions/transcribed/MI.AA.UM.KM.L.878.r.xml

```diff
@@ -132,7 +132,7 @@
         
         
                 <div type="commentary">
-                    <p/>
+                   <p>The inscription on the other is side is MI.AA.UM.KM.L.878.v.</p>
```

Comment:
Do we need to note this? I thought this was assumed based on the file-name.

Reply:
I think I was told at some point to include it in the commentary, but it does seem redundant.